js/components/app.js

```diff
@@ -0,0 +1,36 @@
+/*
+var LoginScreen = require("./loginScreen.js");
```

Comment:
We actually need a bundler like webpack. But I want to keep things simple. So. let's avoid that for a litte while more.